test/500-reverse-lookups.bats

```diff
@@ -22,18 +22,20 @@ load helpers
 	create_container "$a2_config"
 	a2_pid="$CONTAINER_NS_PID"
 
-	echo "$a1_config"
-	echo "$a2_config"
+	echo "a1 config:\n${a1_config}\n"
+	echo "a2 config:\n${a2_config}\n"
 
 	# Resolve IPs to container names
 	dig_reverse "$a1_pid" "$a2_ip" "$gw"
-	assert "$output" =~ "atwo"
-	assert "$output" =~ "a2"
-	assert "$output" =~ "2a"
+	echo -e "Output:\n${output}\n"
+	assert "$output" =~ '([0-9]+\.){4}in-addr\.arpa\.[ 	].*[ 	]atwo\.'
```

Comment:
I think we should make sure that the ip is printed in reverse order.
This regex would allow any ip but we really should make sure it is actually the ip we want.

Maybe for ipv4 we can do `echo $ip | awk -F. '{print $4"."$3"." $2"."$1}'` to invert it, for ipv6 something like this would not really work but I think if we at least properly verify ip4.
For ipv6 our test should always use a fd... adress so you could at least match .d.f.ipv6.arpa

Reply:
Ok, I did what you suggested except I used `printf` and constructed the whole address. For IPv6 I also can't think of anything that would help much so I went with what you proposed.